packages/remix-react/errors.ts

```diff
@@ -3,6 +3,10 @@ import { ErrorResponse } from "@remix-run/router";
 
 import type { AppData } from "./data";
 
+/**
+ * @deprecated in favor of React Router `ErrorResponse`
+ * TODO: Need to add generics in RR
+ */
 export interface ThrownResponse<
```

Comment:
I'm not even sure we need this or the generics once `v2_errorBoundary` becomes the default since we won't have `useCatch` anymore?